modin/core/dataframe/algebra/groupby.py

```diff
@@ -194,7 +210,7 @@ def map(
     @classmethod
     def reduce(
         cls,
-        df,
+        df: pandas.DataFrame,
```

Comment:
Why are other parameters untouched? There are many similar places.

Reply:
For this particular place I missed it, I don’t see others like it.

Reply:
added